### PR TITLE
Blog - How to get started with PyMongo

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
https://6398c0523632557db339e1cd--ephemeral-narwhal-3ac3b2.netlify.app
@jalletto this might have code area errors, please tell me so i can fix when you review

File: blog/_posts/2022-12-31-starting-with-pymongo.md [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+
+~~~{.python caption="main.py"}
+book_id = ObjectId("6383ba5c7c58adf893a71694")
+updates = {
+    "$set": {
+        "author": "Unknown Author"
+    }
+}
+book_collection.update_one(filter={"_id": book_id}, \
+update=updates)
```
Comment (jalletto) [inline blog/_posts/2022-12-31-starting-with-pymongo.md:424]:
```suggestion
book_collection.update_one( filter={"_id": book_id}, 
                                                  update=updates
                                                 )
```

Comment (jalletto) [reply blog/_posts/2022-12-31-starting-with-pymongo.md:424]: @msali123 For function calls in python, we can spread them over multiple lines without needing to use the \.

File: blog/_posts/2022-12-31-starting-with-pymongo.md [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+~~~
+
+> Notice the `$set` operator used in the `updates` dictionary. The operator is used to set the value of a field in MongoDB. You can learn more about Operators here.
+If you want to update more than one document at a time, you can use the `update_many()` method as below:
+
+~~~{.python caption="main.py"}
+book_collection.update_many(filter={}, \
+update={"$set": {"author": "Unknown Author"}})
```
Comment (jalletto) [inline blog/_posts/2022-12-31-starting-with-pymongo.md:432]:
```suggestion
book_collection.update_many(filter={}, 
                                                   update={"$set": {"author": "Unknown Author"}}
                                                   )
```

**Review by jalletto — CHANGES_REQUESTED** [review]

**Review by jalletto — APPROVED** [review]

File: blog/_posts/2022-12-31-starting-with-pymongo.md [hunk 3]
```diff
@@ -0,0 +1,464 @@
...
+                                                 )
+~~~
+
+> Notice the `$set` operator used in the `updates` dictionary. The operator is used to set the value of a field in MongoDB. You can learn more about Operators here.
+If you want to update more than one document at a time, you can use the `update_many()` method as below:
+
+~~~{.python caption="main.py"}
+book_collection.update_many(filter={}, 
+                                                   update={"$set": {"author": "Unknown Author"}}
+                                                   )
```
Comment (jalletto) [inline blog/_posts/2022-12-31-starting-with-pymongo.md:434]:
```suggestion
~~~{.python caption="main.py"}
book_collection.update_many(filter={}, update={"$set": {"author": "Unknown Author"}})
```

File: blog/_posts/2022-12-31-starting-with-pymongo.md [hunk 4]
```diff
@@ -0,0 +1,464 @@
...
+book_id = ObjectId("6383ba5c7c58adf893a71694")
+updates = {
+    "$set": {
+        "author": "Unknown Author"
+    }
+}
+book_collection.update_one( filter={"_id": book_id}, 
+                                                  update=updates
+                                                 )
+~~~
```
Comment (jalletto) [inline blog/_posts/2022-12-31-starting-with-pymongo.md:426]:
```suggestion
book_collection.update_one( filter={"_id": book_id}, update=updates)
~~~
```

**Review by jalletto — APPROVED** [review]

**Pull request merged**